practitioner/src/main/java/org/smartregister/extension/rest/PractitionerDetailsResourceProvider.java

```diff
@@ -430,8 +431,9 @@ private List<String> getLocationIdentifiersByOrganizations(List<Organization> or
                     organizationAffiliationObj = (OrganizationAffiliation) organizationAffiliation;
                     List<Reference> locationList = organizationAffiliationObj.getLocation();
                     for (Reference location : locationList) {
-                        if (location != null && location.getReference() != null) {
-                            locationsIdentifiers.add(location.getReference());
+                        if (location != null && location.getReference() != null && locationsIdentifiers.size() > 0 &&
```

Comment:
Did we mean to add the check `locationsIdentifiers.size() > 0` here? 

As it is right now, looks like nothing will ever get added since the _locationsIdentifiers_ array list is always empty to begin with.

Reply:
Updated